experimental/dds/tree2/src/shared-tree/sharedTree.ts

```diff
@@ -60,6 +86,15 @@ export interface ISharedTree extends ISharedObject, TypedTreeChannel {
 	// TODO: migrate all usages of this to alternatives or avoid using ISharedTree.
 	readonly view: ISharedTreeView;
 
+	/**
+	 * Provides a copy of the current content of the tree.
+	 * This can be useful for inspecting the tree when no suable view schema is available.
+	 * This is only intended for use in testing and exceptional code paths: it is not performant.
+	 *
+	 * This does not include everything that is included in a tree summary, since information about how to merge future edits is omitted.
+	 */
+	contentSnapshot(): TreeSnapshot;
```

Comment:
Do we have other production dependencies on `jsonableTreeFromFieldCursor`? (I know we currently use it for the wire format but intend not to before MVP).

It'd be nice to not need this, but I don't see a super obvious way, and this does seem like a lesser evil to exposing the view.

Reply:
I guess it's not that much code anyway, and I do like the debuggability convenience. Maybe my distaste is misguided :)

Reply:
We have a long-standing API issue of using arrays of nodes cursors instead of field cursors in lots of places (ex: all content insertion APIs, deltas etc) which is currently wrecking our compression efficiency, so moving toward easier use of field cursors is good for resolving that.

I'm not sure why this comment is on this line. Maybe you are asking about exposure of the jsonable tree format in general?

Anyway, I'm not intending for this to be included in our MVP APIs, but we don't have an approved process from the API review council for how to how we can separate our current rather arbitrary package exported surface from our actual intended MVP package API. I hope to be able to do that with @beta, but for now it's all in my head and I can say that this interface is not part of that surface (ISharedTreeView and editiable-tree-2 are also exposed by this interface and not part of the planned MVP package API surface)

Reply:
Yeah, I was wondering from a bundle size point of view in addition to the API concern.

But this is extra minor if this API isn't even part of planned MVP surface. Thanks for the context.